### PR TITLE
chore(apps/interface): refactor IndexPage imports

### PR DESCRIPTION
**Pull request by asaadam**
Copy paste the following markdown in the pull request

## Scope
List of affected projects:

- apps/interface

## Description
We need to use default import for all imported components and utilities in [IndePage][1].

[1]: https://github.com/risedle/monorepo/blob/main/apps/interface/pages/index.tsx#L4-L15

## Action items
Action items for this pull request:

- [X] Update `import {x} from "y"` to `import x from y`

## Checklist
You should check this before requesting for review:

- [X] Branch name use the the following format RIS-{NUMBER}
- [X] Pull request title is "chore(apps/interface): refactor IndexPage imports"
- [X] Make sure new files are 100% covered in [Risedle Code Coverage](https://coverage.risedle.com)

### CODE REVIEW

File: apps/interface/components/HomeHeading/index.tsx [hunk 1]
```diff
@@ -111,3 +111,5 @@ export const HomeHeading = (props: HomeHeadingProps) => {
         </Container>
     );
 };
+
+export default HomeHeading;
```
Comment (pyk) [inline apps/interface/components/HomeHeading/index.tsx:115]: We need to remove the individual export in this component.

Replace:

```ts
export const HomeHeading = ...
```

with:

```
const HomeHeading = ...
```

File: apps/interface/components/TokenCard/cards.tsx [hunk 2]
```diff
@@ -26,3 +26,5 @@ export const TokenCards = (props: TokenCardsProps) => {
         </Container>
     );
 };
+
+export default TokenCards;
```
Comment (pyk) [inline apps/interface/components/TokenCard/cards.tsx:30]: We need to remove the individual export in this component.

Replace:

```ts
export const TokenCards = ...
```

with:

```
const TokenCards = ...
```

File: apps/interface/components/WarningBar/index.tsx [hunk 3]
```diff
@@ -110,3 +110,5 @@ export const WarningBar = () => {
         </Container>
     );
 };
+
+export default WarningBar;
```
Comment (pyk) [inline apps/interface/components/WarningBar/index.tsx:114]: We need to remove the individual export in this component.

Replace:

```ts
export const WarningBar = ...
```

with:

```ts
const WarningBar = ...
```

File: apps/interface/utils/fetchFuseLeveragedTokens.ts [hunk 4]
```diff
@@ -79,3 +79,5 @@ export const fetchFuseLeveragedTokens =
         const { graphEndpoint } = getBaseConfig();
         return await grequest(graphEndpoint, queryFuseLeveragedTokens);
     };
+
+export default fetchFuseLeveragedTokens;
```
Comment (pyk) [inline apps/interface/utils/fetchFuseLeveragedTokens.ts:83]: We need to remove the individual export in this utils.

Replace:

```ts
export const fetchFuseLeveragedTokens = ...
```

with:

```ts
const fetchFuseLeveragedTokens = ...
```

File: apps/interface/utils/getFuseLeveragedTokensSummary.ts [hunk 5]
```diff
@@ -20,3 +20,4 @@ export const getFuseLeveragedTokensSummary = (
         };
     }, initial);
 };
+export default getFuseLeveragedTokensSummary;
```
Comment (pyk) [inline apps/interface/utils/getFuseLeveragedTokensSummary.ts:23]: We need to add whitepsace above this line

Then Replace:

```ts
export const getFuseLeveragedTokensSummary = ...
```

with:

```ts
const getFuseLeveragedTokensSummary = ...
```

**Review by pyk — CHANGES_REQUESTED** [review]
Just smol nits @asaadam

**Comment by asaadam** [comment]
@pyk if we do that, we should change the test file too, it's that okay?

**Comment by pyk** [comment]
@asaadam yes lets update the imports in the test too

**Review by pyk — APPROVED** [review]
Thanks @asaadam !

**Pull request merged**